### PR TITLE
Refactor/challenge launch

### PR DESCRIPTION
**Pull request by reinzor**
Action server launch file deleted; action server is now treated exactly the same as a challenge. Also added rqt launch file for challenge GUI and some respawn settings to make everythin close nicely.

### CODE REVIEW

**Comment by LoyVanBeek** [comment]
So before you use the app to send actions, you must start the action server explicitly, via the robot? 
That does not make for a more n00b friendly robot

**Comment by reinzor** [comment]
So just to be clear, the action server is just another state machine, instead of a challenge. I will make a GUI to start either the "free_Mode" action server state machine or a challenge and make everything nicely close each other.

File: launch/interaction/action_server.launch [hunk 1]
```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0"?>
```
Comment (LoyVanBeek) [inline launch/interaction/action_server.launch:1]: There is no replacement launch file in [action_server](https://github.com/tue-robotics/action_server) yet

**Review by LoyVanBeek — APPROVED** [review]

File: launch/interaction/action_server.launch (hunk 1 above)
Comment (reinzor) [reply launch/interaction/action_server.launch:1]: The won't be any, action server will be treated as a challenge

**Pull request merged**